src/comparsion.py

```diff
@@ -0,0 +1,161 @@
+import pandas as pd
+pd.set_option('display.max_columns', None)
+pd.set_option('display.width', 1000)
+
+import json
+import re
+from pathlib import Path
+
+# orginally clojure_mapping
+combined_mapping = {
+    'pre-increment': 'preIncr',
+    'operator-precedence': 'operator_precedence',
+    'conditional': 'conditional',
+    'preprocessor-in-statement': 'preprocessor_in_statement',
+    'logic-as-control-flow': 'logic_as_control_flow',
+    'type-conversion': 'type_conversion',
+    'comma-operator': 'comma_atoms',
+    'implicit-predicate': 'implicit_predicate',
+    'post-increment': 'postIncr',
+    'assignment-as-value': 'assignment_as_value',
+    'repurposed-variable': 'repurposed_variable',
+    'omitted-curly-braces' : 'omitted_curly_braces',
+    'macro-operator-precedence' : 'macro_operator_precedence',
+    'literal-encoding' : 'literal_encoding'
+}
+
+CLJnCQL = ["CLJ", "CQL"]
+CLJnCNL = ["CLJ", "CNL"]
+CQLnCNL = ["CQL", "CNL"]
+all = ["CLJ", "CQL", "CNL"]
+
+def count_plus_specific_columns(row, columns):
+    return (row[columns] == '+').sum()
+
+def compare_and_output(output_directory, codeql_system_path, clojure_system_path, coccinelle_system_path, whichtwo, codeQL_directory, clojure_directory, coccinelle_directory):
+    print(whichtwo)
+    dataframes = {}
+    frame = pd.DataFrame(columns= ["Type", "File", "Line"])
+    dataframes["preIncr"] = frame
+    dataframes["operator_precedence"] = frame
+    dataframes["conditional"] = frame
+    dataframes["preprocessor_in_statement"] = frame
+    dataframes["logic_as_control_flow"] = frame
+    dataframes["type_conversion"] = frame
+    dataframes["comma_atoms"] = frame
+    dataframes["implicit_predicate"] = frame
+    dataframes["postIncr"] = frame
+    dataframes["assignment_as_value"] = frame
+    dataframes["repurposed_variable"] = frame
+
+    which_two = []
+
+    if whichtwo == "CQLvCNL":
+        which_two = CQLnCNL
+    elif whichtwo == "CLJvCQL":
+        which_two = CLJnCQL
+    elif whichtwo == "CLJvCNL":
+        which_two = CLJnCNL
+    else:
+        which_two = all
+
+    # Load CodeQL .sarif files and store them in dataframes dictionary
+
+    if whichtwo == "CQLvCNL" or whichtwo == "CLJvCQL" or whichtwo == "all":
+        for file_path in Path(codeQL_directory).glob('*.sarif'):
+            file_content = file_path.read_text()
+            filename = Path(file_path).stem
+            dataframe_key = filename
+            json_value = json.loads(file_content)
+            p = pd.DataFrame(["Type", "File", "Line", "CQL", "CQL_Column", "CQL_CodeQL"])
```

Comment:
This code is still in `atom_comparison_tool.py` too, right?  If `atom_comparison_tool.py` is not longer needed, let's remove it